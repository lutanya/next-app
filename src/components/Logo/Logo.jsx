import React from 'react';

export default function Logo() {
  return (
    <>
      <div>
        <b>netflix</b>
        roulette
      </div>
      <style jsx>
        {`
        div {
            color: #F65261;
            font-size: 1.1em;
        }`}
      </style>
    </>
  );
}
