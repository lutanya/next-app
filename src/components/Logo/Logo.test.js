import React from 'react';
import {render} from '@testing-library/react';
import {Logo} from './Logo';

describe('Logo',()=>{
    test('Render Logo snapshot', ()=>{
        const {asFragment} = render(<Logo/>) ;
        expect(asFragment(<Logo/>)).toMatchSnapshot();
    })
})

