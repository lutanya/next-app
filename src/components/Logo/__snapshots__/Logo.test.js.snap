// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Render Logo snapshot 1`] = `
<DocumentFragment>
  <div
    class="logo"
  >
    <b>
      netflix
    </b>
    roulette
  </div>
</DocumentFragment>
`;
